tests/integration/test_envelope.py

```diff
@@ -579,3 +579,51 @@ def test_sample_rates_metrics(mini_sentry, relay_with_processing, events_consume
 
     event, _ = events_consumer.get_event()
     assert event["_metrics"]["sample_rates"] == sample_rates
+
+
+# Checks that we buffer envelopes until we have a global config in processing mode.
+def test_buffer_envelopes_without_global_config(
+    mini_sentry, relay_with_processing, events_consumer
+):
+    include_global = False
+    original_endpoint = mini_sentry.app.view_functions["get_project_config"]
+
+    @mini_sentry.app.endpoint("get_project_config")
+    def get_project_config():
+        nonlocal include_global
+
+        res = original_endpoint().get_json()
+        if not include_global:
+            res.pop("global", None)
+        return res
+
+    mini_sentry.add_basic_project_config(42)
+    events_consumer = events_consumer()
+    options = {"cache": {"global_config_fetch_interval": 1}}
```

Comment:
If we set the value to `0`, does it speed the test up? Does this enable reducing the wait time at the bottom of the test?

Reply:
yeah thanks! 

i had forgotten to reduce the 10 sec interval after i reduced the fetch_interval, it would work with 2 sec timeout but now it works with 1 sec after putting the fetch_interval to 0 ^^